issue9/src/issue9.ts

```diff
@@ -0,0 +1,37 @@
+//Function to add a passed number objects to passed array
+function addToArray(array: {name: string, address: string, phone: string} [], inputNumber: number) {
+    for (let x = 0; x < inputNumber; x++) {
+        array.push({
+            "name": makeRandomString(8),
+            "address": makeRandomString(8),
+            "phone": makeRandomString(8)
+        });
+    }
+    return array;
+};
+
+//Function to generate string of random characters of given length
+function makeRandomString(length: number) {
+    let randomString = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    while (randomString.length < length) {
+        randomString += characters.charAt(Math.floor(Math.random() * characters.length));
+    }
+    return randomString;
+};
+
+//Type for an array of objects with values of strings
+const objArray: {name: string, address: string, phone: string} [] = [
+    {"name": "John Doe", "address": "101 SomeStreet", "phone": "5555555555"}];
+
+//Call of function to add objects to array
+console.log(addToArray(objArray, 5));
+
+//Created empty array with array type 'object'
+const newObjArray: {name: string, address: string, phone: string} [] = [];
```

Comment:
I see that you're doing all these types inline. However, you're losing one of the benefits of typescript and that's consistency.

Declare a type at the top as `{name: string, address: string, phone: string} []` and reference it in all these other places.

Reply:
I see what you mean, and that makes sense. I'll make the change

Reply:
made changes.